gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,137 @@
 ---
-description: Proof Market front end guide
+description: The web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
 
 {% hint style="info" %}
 Please note this product is in beta and expected to change.
 {% endhint %}
 
-#### Proof Market front end can be accessed at [**proof.market**](https://proof.market)
+#### The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
 
 ## New user signup
```

Comment:
First, only new users sign up, the non-new ones log in. Second, let's give it a better name.

```suggestion
## Creating an account
```

Reply:
fixed